test-utils/junit5-extension-dmn/pom.xml

```
@@ -0,0 +1,71 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>camunda-dmn-junit5</artifactId>
+  <packaging>jar</packaging>
+
+  <name>Camunda DMN - JUnit 5 Extension</name>
+
+  <parent>
+    <groupId>org.camunda.bpm.dmn</groupId>
+    <artifactId>camunda-engine-dmn-root</artifactId>
+    <version>7.20.0-SNAPSHOT</version>
+    <relativePath>../engine-dmn</relativePath>
+  </parent>
```

Let's use the database parent like the BPM extension does to stay consistent with the hierarchy.

```suggestion
  <parent>
    <groupId>org.camunda.bpm</groupId>
    <artifactId>camunda-database-settings</artifactId>
    <version>7.20.0-SNAPSHOT</version>
    <relativePath>../../database</relativePath>
  </parent>
```